lib/contentful-management.ts

```diff
@@ -61,14 +65,25 @@ function createClient(
     defaults?: DefaultParams
   }
 ): PlainClientAPI
+function createClient<
+  T extends (ReadonlyArray<string> | readonly ['workflows']) &
+    { [K in keyof T]: { [P in K]: 'workflows' } }[number]
+>(
```

Comment:
This describes an array that contains "workflows" somewhere in the list.

Reply:
Sounds like this should be a comment in code :p

Reply:
Generally, this is very unclear to me. Could you please define a private type for it and name it so it defines the result. If it's a map or a string or an array. Too many layered nesting in types 😓